project-base/storefront/components/Blocks/StoreList/StoresWrapper.tsx

```diff
@@ -27,11 +51,25 @@ export const StoresWrapper: FC = () => {
     return (
         <div className="flex w-full flex-col lg:flex-row lg:gap-5">
             <div className="w-full max-lg:order-2 max-lg:mt-5 lg:basis-1/2">
+                <div className="mb-2.5">
+                    <SearchInput
```

Comment:
you can pass `className="mb-2.5"` to the `SearchInput` after that you can remove wrapper div right above

Reply:
I cant. This will add that to the input and not the wrapper div

Reply:
ok then :)